test/functional/feature_backwards_compatibility.py

```diff
@@ -331,6 +340,29 @@ def run_test(self):
         hdkeypath = v17_info["hdkeypath"]
         pubkey = v17_info["pubkey"]
 
+        # Copy the 0.16 wallet to the last Bitcoin Core version and open it:
```

Comment:
In commit "tests: feature_backwards_compatibility.py test 0.16 up/downgrade" (7a872702965755ce28f1f17a8d0e0eb93ae4c907)

Implemention seems ideal for this PR, but for future cleanup it could be nice for someone to write some python helper functions and get rid of duplicate code in the 0.16, 0.17, and 0.19 test cases.

Reply:
Indeed, I've been kicking that down the road too, see #19013